### PR TITLE
created video library basic structure and started watchlater feature

### PR DESCRIPTION
**Pull request by sakshamAK**
completed pages
 - trending
 - home
 - signin 
 - signup
 - watchlater
 - explore
 - playlists
 - liked videos
 - history

**HOME**
image

**TRENDING**
image

**SIGN IN**
image

**SIGN UP**
image

**EXPLORE, PLAYLISTS, LIKED VIDEOS, WATCH LATER**
image

### CODE REVIEW

File: src/pages/Home/Home.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+            <CoverContent />
+            <div className="videoListing">
+                <h1 className={`${styles.heading}`} >Categories</h1>
+                <div className="videoContainer">
+                    {categoryData.map(({ _id, thumbnail, category, title, views, creator }) => (
+                        <Link className={`${styles.productLink}`} to={`/${category}`} key={_id}>
+                            <div className={`${styles.thumbnailContainer}`}>
+                                <img className="img-resp" src={thumbnail} alt={category} title={title} />
+                                <h4>{title}</h4>
+                                {category !== "" ? <h4>{category}</h4> : ""}
```
Comment (mimansha-swarup) [inline src/pages/Home/Home.js:22]: suggestion you can use  && instead of ternary operator

**Review by mimansha-swarup — APPROVED** [review]
everything else looks great
clean and readable code
good folder structure
nice UI

File: src/pages/Home/Home.js (hunk 1 above)
Comment (BelideNikhil) [reply src/pages/Home/Home.js:22]: Using ternary is better than &&  also, you can return ```null``` instead of empty string

**Review by BelideNikhil — APPROVED** [review]
The code is really well written and UI looks and works really nicely
One suggestions to return null instead of empty string when using ternary to display components

**Review by mimansha-swarup — CHANGES_REQUESTED** [review]
Readable code
All looks ok 
Good to go 👍

**Review by mimansha-swarup — APPROVED** [review]

**Pull request merged**